source/configuring-ember/preparing-for-upgrade.md

```diff
@@ -0,0 +1,154 @@
+A valueable attribute of the EmberJS framework is its use of [Semantic Versioning](http://semver.org/), that allows projects to keep up with
+changes to the framework.  Before any function or API is removed it first goes through a deprecation period, where the the functionality is
+still supported, but usage of it generates a warning.  These warnings can pile up between major releases to a point where the amount of
+deprecation warnings that scroll through the console becomes overwhelming.
+
+![Deprecations Clouding up the Browser JavaScript Console](../../images/guides/configuring-ember/preparing-for-upgrade/deprecations-in-console.png)
+
+Fortunately, Ember provides a systematic way for projects to deal with deprecations in an organized and efficient manner.
```

Comment:
will probably remove "systematic" because it feel redundant to the last part of the sentence.